posthog/models/filters/mixins/session_recordings.py

```diff
@@ -21,3 +25,23 @@ def recording_duration_filter(self) -> Optional[Property]:
             filter_data = json.loads(duration_filter_data_str)
             return Property(**filter_data)
         return None
+
+
+class SessionRecordingsMetadataMixin(BaseParamMixin):
```

Comment:
I don't think this makes sense. **Filters** is for filtering. `include_metadata_for_recordings` is just a query param for the new endpoint so we should just use it that way like any other query param